pkg/manager/member/config.go

```diff
@@ -29,6 +31,8 @@ func updateConfigMap(old, new *corev1.ConfigMap) (bool, error) {
 
 	// check config
 	tomlField := []string{"config-file" /*pd,tikv,tidb */, "pump-config", "config_templ.toml" /*tiflash*/, "proxy_templ.toml" /*tiflash*/}
+	yamlField := []string{"prometheus-config", "prometheus.yml" /*prometheus*/}
```

Comment:
`prometheus-config` is used as a key for external configmap that can be mounted for Prometheus which is not managed by TiDB Operator, so we can ignore this.
However, `dashboards.yaml` should be considered.

Reply:
OK, fixed.